app/upgrades/v8_2/constants.go

```diff
@@ -0,0 +1,10 @@
+package v82
+
+const (
+	// UpgradeName is the shared upgrade plan name for mainnet and testnet
```

Comment:
nit, you mentioned that this upgrade shouldn't be used in testnet
```suggestion
	// UpgradeName is the shared upgrade plan name for mainnet
```